vyper/compiler/output.py

```diff
@@ -132,7 +134,19 @@ def _compress_source_map(code, pos_map, jump_map, source_id):
     return compressed_map
 
 
+# EIP-170. Ref: https://eips.ethereum.org/EIPS/eip-170
+EIP_CONTRACT_SIZE_LIMIT: int = 2 ** 14 + 2 ** 13
+
+
 def build_bytecode_output(compiler_data: CompilerData) -> str:
```

Comment:
This logic belongs in `build_bytecode_runtime_output` I believe. The limit isn't applied to the creation bytecode, only to what gets deployed on-chain.